README.md

```diff
@@ -91,6 +91,7 @@ render(
   markdown,
   smart-punctuation: true,
   blockquote: none,
+  mitex: none,
```

Comment:
Can we rename this option to `math`? Hypothetically it can be used with backends other than MiTeX.